ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.h

```diff
@@ -3,6 +3,7 @@
 
 #define BOARD_FLASH_SIZE (8 * 1024 * 1024)
 
-#define MICROPY_HW_LED_PINMUX  IOMUXC_GPIO_AD_B0_09_GPIO1_IO09
-#define MICROPY_HW_LED_PORT    GPIO1
-#define MICROPY_HW_LED_PIN     9
+// MIMXRT1060_EVK has 1 user LED
+#define MICROPY_HW_LED1_PIN (GPIO_AD_B0_09)
+#define MICROPY_HW_LED_ON(pin) (mp_hal_pin_high(pin))
```

Comment:
According to my understanding of the schematics the user LED is pulled up to 3.3V supply. To switch it on pin should be pulled low I believe.